tests/integration/cli/test_help.py

```diff
@@ -11,11 +14,18 @@ def test_help_page_for_non_aliased_actions():
     output = process.stdout.decode()
     wrapped_output = textwrap.fill(output, width=150).replace("\n", "")
 
-    assert "Linodes List" in wrapped_output
-    assert (
-        "API Documentation:  https://www.linode.com/docs/api/linode-instances/#linodes-list"
-        in wrapped_output
+    assert contains_at_least_one_of(
+        wrapped_output, ["Linodes List", "List Linodes"]
     )
+
+    assert contains_at_least_one_of(
+        wrapped_output,
+        [
+            "API Documentation:  https://www.linode.com/docs/api/linode-instances/#linodes-list",
+            "API Documentation:  https://www.linode.com/docs/api/linode-instances/#list-linodes",
```

Comment:
There are some minor (but valid) changes to the help pages/descriptions for certain endpoint methods. I updated the tests accordingly to ensure everything will be backwards compatible